posthog/models/person/person.py

```diff
@@ -99,3 +99,24 @@ class Meta:
 
     # current version of the id, used to sync with ClickHouse and collapse rows correctly for new clickhouse table
     version: models.BigIntegerField = models.BigIntegerField(null=True, blank=True)
+
+
+class PersonOverride(models.Model):
+    """A model of persons to be overriden in merge or merge-like events."""
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["old_person_id", "override_person_id"], name="unique override per old_person_id"
+            )
+        ]
+
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+
+    # We don't want to delete rows before we had a chance to propagate updates to the events table.
+    # To reduce potential side-effects, these are not ForeingKeys.
+    old_person_id = models.UUIDField(db_index=True)
+    override_person_id = models.UUIDField(db_index=True)
+
+    created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True, blank=True)
```

Comment:
this is the old_person created_at, correct? please add a comment that's not obvious. Also the motivation for this field to know whether this will be a short or long term override in CH.

Reply:
Instead of a comment I could rename the field to `old_person_created_at` to dispel all doubts.

Reply:
Also, I believe I missed `is_historical`/`is_longterm` flag we talked about...

Reply:
Yes, thought this through a bit more we'll want the flag. This would make it easier to sync from postgres to clickhouse if they got out of sync, as postgres is the source of truth here.

One could think that we can just use `old_person_created_at` value to figure out which table it should go to, but we can't because we don't know at which time the entry was added to postgres. Hence if we had a 45 day cut off and we see `old_person_created_at` that's 46 days ago it might have been added today (long term table) or it might have been added 2 days ago (short term table).

Reply:
Do we need the flag then or a `override_added_at` field?

Reply:
I'm pushing a commit to compute the flag in a second, fighting with some unit tests...

Reply:
Ok, gave it a shot. The `is_long_term` flag requires `old_person_created_at` in order to compare whether that is past the cut off point from now. Let me know what you think.

Reply:
We could also look up the `old_person_created_at` ourselves, thus not requiring that parameter. As the one who will be implementing the ingestion of these tables, what do you think?